test/system/conversations/messages/images_test.rb

```diff
@@ -103,15 +103,14 @@ class ConversationMessagesImagesTest < ApplicationSystemTestCase
       refute img.visible?
 
       image_btn.click
-
       2.times do
         sleep 0.1
         sleep 0.5 if !modal_loader.visible?
         sleep 0.1
         image_btn.click if !modal_loader.visible?
       end # TODO: sometimes modal has not popped up after clicking, why?? Try 2x times before failing the test.
```

Comment:
@jlvallelonga I think we had 3 distinct test failures. You fixed one of them with the update to `simulate_image_variant_processing`. I tried to fix a sporadic `Net::ReadTimeout` with a timeout. However, there remains one more test failure on this line. I think it might be a legitimate bug.

When I turn off headless chrome and run just this specific test, I do not see the spinners in place of the images. I spent a little while investigating it and couldn't figure out why not. The thing I did not try was checking out an old commit right before the refactor to image attachments, and that would confirm for sure that we broke it recently.

Could you do a little more investigating of this one? What you should see when running `bin/rails test test/system/conversations/messages/images_test.rb:88` in normal chrome mode is the spinners appearing for 8 seconds both inline in the conversation messages and also when you click on a spinner you see a spinner within the modal that opens. Then after 8 seconds (of faked delay) then the spinners are swapped out for the images.

Reply:
ok nice! yeah I'll dig into this last failure tomorrow